src/main/webapp/script.js

```diff
@@ -0,0 +1,40 @@
+function loadBody() {
+  verifyLogin();
+}
+
+/** Contacts Login servlet to verify if user is logged in. */
+function verifyLogin() {
+  fetch('/login').then(response => response.json()).then((userDetails => {
+    const loginElement = document.getElementById("login-box");
+    const schoolName = document.getElementById("school-name");
+    if (!userDetails.loggedIn) {
+      schoolName.style.display = "none";
+      loginElement.innerHTML += "<p><a href=\"" + userDetails.url + "\"> Click here to log in! </a></p>";
+    } else {
+      schoolName.style.display = "block";
+      schoolName.innerHTML += userDetails.schoolName;
+      loginElement.innerHTML += "<p><a href=\"" + userDetails.url + "\"> Click here to log out! </a></p>";
+    }
+  }));
+}
+
+function showClasses() {
+  const classes = document.getElementById("search-results");
+  classes.innerHTML = "";
+  fetch('/search')
+  .then(response => response.json())
+  .then((response) => {
+    return JSON.parse(response.classNames);
+  })
+  .then((classNames) => {
+    console.log(classNames);
+    classNames.forEach(name => createListElement(name));
```

Comment:
This doesn't actually insert the list elements

Reply:
Realized that - I changed it to append the element to classNames, but I think I committed that in the search branch instead

Reply:
This won't block this time around but it's good practice to have different branches for different features you're working on.